multiregion/README.md

```diff
@@ -276,7 +276,7 @@ Stop the demo and all Docker containers.
 
 ## Troubleshooting
 
-1. If containers fail to ping each other (failures seen in running `./scripts/validate_connectivity.sh`), clean up the Docker environment before running the demo again.
+1. If containers fail to ping each other (e.g., failures seen in running `./scripts/validate_connectivity.sh`), clean up the Docker environment and then run the demo again.
```

Comment:
maybe suggest a command for "clean up the Docker environment"?

Reply:
@rspurgeon it's not rendering in the diff, but the command is shown right after this line

Reply:
doh! I see it now...   Now that I do, the commands do worry me a bit in the sense that they forecefully destroy docker resources across the board (including data volumes which might have value to the user).   Maybe we should be more specific about cleaning up only our resources by name or id to prevent accidental deletion of our end users resources?

Reply:
@rspurgeon Agree!  I'll file a follow-on jira to filter on resources.